apps/admin-panel/src/components/Sidebar/SidebarContainer.tsx

```
@@ -10,6 +11,200 @@ import {
+        }
+        return false;
+      });
+
+    const active = Boolean(topMatches || childrenMatch);
+
+    if (item.children && item.children.length) {
+      return (
+        <div
+          onMouseEnter={(e) => handleFlyoutEnter(item, e)}
+          onMouseLeave={handleFlyoutLeave}
+        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)